packages/engine/Source/Shaders/SkyAtmosphereFS.glsl

```diff
@@ -42,14 +43,7 @@ void main (void)
     #endif
 
     #ifdef COLOR_CORRECT
-        // Convert rgb color to hsb
-        vec3 hsb = czm_RGBToHSB(color.rgb);
-        // Perform hsb shift
-        hsb.x += u_hsbShift.x; // hue
-        hsb.y = clamp(hsb.y + u_hsbShift.y, 0.0, 1.0); // saturation
-        hsb.z = hsb.z > czm_epsilon7 ? hsb.z + u_hsbShift.z : 0.0; // brightness
```

Comment:
@ggetz or @lilleyse is there any reason the old code doesn't allow brightness shifts less than epsilon 7? Most notably, why aren't negative values allowed?

I'm assuming not, so when making `czm_applyHSBShift()` I implemented this as

`hsb.z = clamp(hsb.z + u_hsbShift.z, 0.0, 1.0);` similar to saturation, that works fine even for negative values.

Reply:
None that I know of.